pkg/asset/installconfig/aws/permissions.go

```diff
@@ -324,6 +327,12 @@ var permissions = map[PermissionGroup][]string{
 		// Needed to filter zones by instance type
 		"ec2:DescribeInstanceTypeOfferings",
 	},
+	PermissionCreateEdgeCompute: {
```

Comment:
@r4f4 do you think we could create a (sub-)group "EdgeWavelength" as it applies only for a subset of edge zones?

If user wants to deploy edge Local Zones it isn't required.

Reply:
If so, to check if without api calls would be by regex expressions against zone names.

Reply:
Yes. Carrier Gateway is applied only for Wavelength Zones.

Reply:
Ops, then yes, the group should be `PermissionCreateEdgeWavelength` or something like that and we need to change the permission checker to look for wavelength zones.